src/topp/IDMapper.cpp

```diff
@@ -92,7 +94,12 @@ using namespace std;
 
     <B>Annotation of consensus maps (consensusXML input):</B>\n
     Peptide positions are always matched against centroid positions. By default, the consensus centroids are used. However, if @p consensus:use_subelements is set, the centroids of sub-features are considered instead.
-    In this case, a peptide identification is mapped to a consensus feature if any of its sub-features matches.
+    In this case, a peptide identification is mapped to a consensus feature if any of its sub-features matches. In the case of consensusXMLs from TMT data, generated by
+    IsobaricAnalyzer, this tool will use the spectrum ID attached to the feature to retrieve an identification from the idXML.
+    
+    <B>Annotation with mzML (both feature and consensusXML input):</B>\n
+    Instead of/in addition to annotating with putatively identified spectra (i.e. PSMs) from idXML, you can use the original spectra in an mzML to attach
+    all potential MS2 spectra to a (Consensus)Feature. See the @p spectra:in option.
```

Comment:
```suggestion
    all potential MS2 spectra to a (Consensus)Feature. See the @p spectra:in option. This can be used for advanced workflows that want to create spectral libraries or targeted assays for unknown entities.
```